### PR TITLE
Improve the guides

### PR DESCRIPTION
**Pull request by chasenlehara**
- Add a note about a bug in `cordova-ios`
- Fix a bunch of links (and convert them to https)
- NPM -> npm
- Mac OS X -> macOS

### CODE REVIEW

File: docs/guides/guide.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Update `src/index.stache` to look like this:
 > New APIs Used:
 > - [<can-import>](https://canjs.com/doc/can-view-import.html) — specifies template dependencies.
 
-If you kept your browser window open at [http://localhost:8080/](localhost:8080) you should see the updated styles and content as soon as you save the file.
+If you kept your browser window open at [http://localhost:8080/] you should see the updated styles and content as soon as you save the file.
```
Comment (imaustink) [inline docs/guides/guide.md:107]: Shouldn't this be a link?

File: docs/guides/place-my-order.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ DoneJS comes with its own server, which hosts your development files and takes c
 donejs develop
 ```
 
-The default port is 8080, so if we now go to [http://localhost:8080/](localhost:8080) we can see our application with a default homepage. If we change `src/index.stache` or `src/app.js` all changes will show up right away in the browser. Try it by changing the `message` property in `src/app.js`.
+The default port is 8080, so if we now go to [http://localhost:8080/] we can see our application with a default homepage. If we change `src/index.stache` or `src/app.js` all changes will show up right away in the browser. Try it by changing the `message` property in `src/app.js`.
```
Comment (imaustink) [inline docs/guides/place-my-order.md:101]: Same here?

File: docs/guides/setting-up.md [hunk 3]
```diff
@@ -110,7 +110,8 @@ When you start Android Studio for the first time it will go through the installa
 
 Once this is complete you should have everything you need for Android development. You can close Android Studio as you won't need it for DoneJS development.
 
-## Mac OS X
+<a id="mac-os-x"></a>
```
Comment (imaustink) [inline docs/guides/setting-up.md:113]: What is this for?

File: docs/guides/guide.md (hunk 1 above)
Comment (chasenlehara) [reply docs/guides/guide.md:107]: It is; you don’t have to include the URL if it’s the same as the link text.

File: docs/guides/setting-up.md (hunk 3 above)
Comment (chasenlehara) [reply docs/guides/setting-up.md:113]: The title changed, so the ID of the section changed, so this is to keep links pointing to the old section ID working.

**Review by imaustink — APPROVED** [review]

**Pull request merged**